ios/Podfile

```diff
@@ -42,6 +42,7 @@ target 'Rainbow' do
 
   pod 'FLAnimatedImage'
   pod 'libwebp'
+  pod 'Kingfisher'
```

Comment:
Because we're using `react-native-fast-image`, we already have [SDWebImage](https://github.com/SDWebImage/SDWebImage) available in the project, so we should use that instead of installing a new dependency.

In order to do that  you should have to:

- Remove the Kingfisher pod
- Add `#import <SDWebImage/SDWebImage.h>` to `Rainbow-Bridging-Header.h`
- Switch the the way the image is loaded [here](https://github.com/rainbow-me/rainbow/pull/352/files#diff-32c0711b36ed5ae7ebff4f098f9d72a8R58) from:
`self.walletImage.kf.setImage(with: url)` to 
`self.walletImage.sd_setImage(with: url)`